### ISSUE TITLE
how to count the extra tex source

### ISSUE THREAD

**Issue by luckystar1992** [EinarAndreas/TeXcount#1]
If there are some extra tex source load by `\input` or `\include`, how to make it work.

**Comment by EinarAndreas** [comment]
On 16/01/2018 02:31, Zheng yuanchun
      wrote:
    
    
      
      If there are some extra tex source load by \input
        or \include, how to make it work.
      
        
          
          
          
          
        
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/EinarAndreas/TeXcount","title":"EinarAndreas/TeXcount","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/EinarAndreas/TeXcount"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"how to count the extra tex source (#1)"}],"action":{"name":"View Issue","url":"https://github.com/EinarAndreas/TeXcount/issues/1"}}}
    
    
    There are two possible command line options you can run TeXcount
    with to do this: -inc or -merge.
    If you use the option -inc, TeXcount will identify included
      source files and analyse these as well. Each file is then analysed
      separately, and at the end a total count is produced.
    Alternatively, if you use the -merge option, TeXcount will merge
      the document together and analyse it as one big document.
    Bibliographies are by default not included even if -inc or -merge
      is specified, but can be added through the option -incbib.
      However, bibliographies are hard to parse for TeXcount and may
      produce unreliable results.
    
    You can get an overview of options from the documentation, the
      web page (http://app.uio.no/ifi/texcount/documentation.html), or
      by running TeXcount with the option --help-options (or --help to
      get an overview of help alternatives).
    
    Einar
